addon/index.js

```diff
@@ -1,7 +1,23 @@
 import Ember from 'ember';
 
-const { isNone } = Ember;
+const { isNone, warn } = Ember;
 
 export default function testSelector(key, value) {
   return isNone(value) ? `[data-test-${key}]` : `[data-test-${key}="${value}"]`;
 }
+
+export function ts(literals, ...substitutions) {
+  let interpolated = literals[0] + substitutions.map((_, i) => substitutions[i] + literals[i + 1]).join('');
+  if (interpolated.indexOf('%') === -1) {
+    warn(`[ember-test-selectors] "${interpolated}" contains no test selectors. Please use the % prefix for test selectors (e.g. ts\`%foo\`).`, false, {
+      id: 'ember-test-selectors.missing-test-selector-prefix',
+    });
```

Comment:
it's not necessary, but can indicate a bug if people forget to include the `%` prefix. e.g. using ``ts`submit` `` instead of ``ts`%submit` ``